internal/ingress/controller/template/template.go

```diff
@@ -762,6 +763,23 @@ type ingressInformation struct {
 	Annotations map[string]string
 }
 
+func (info *ingressInformation) compare(other *ingressInformation) bool {
```

Comment:
Please rename to `Equal` (to be consistent)

Reply:
Done. Thanks for reviewing.